frontend_v2/src/workspace/button.js

```diff
@@ -0,0 +1,54 @@
+import { Text, Container, Graphics } from 'pixi.js';
+
+/////////////////////////////////////////////////////////////
+
+/**
+ * Create a button formed by a Text graphics plus a rounded rectangular background
+ */
+export const button = (text, x, y, buttonColor = 'black', fontColor = 'white', isLeftAligned = true) => {
```

Comment:
For instructional purposes, I think it is better to extend the `Container` class, as we currently do.